core/modules/file/file.module

```diff
@@ -1621,6 +1645,113 @@ function file_managed_file_pre_render($element) {
   return $element;
 }
 
+/**
+ * AJAX callback for managed_file elements to open a browser modal dialog.
+ *
+ * @see file_managed_file_process()
+ */
+function file_managed_file_browser_open($form, $form_state) {
+  // Get the parent "managed_file" element.
+  $parents = $form_state['triggering_element']['#array_parents'];
+  array_pop($parents);
+  $element = backdrop_array_get_nested_value($form, $parents);
+
+  // Pull the view to open off of the managed_file #browser_view property.
+  $view_name = $element['#browser_view'];
+  $view = views_embed_view($view_name);
+
+  $submit_form = backdrop_get_form('file_managed_file_browser_form');
+
+  $title = t('Select from Image Library');
+  $options = array(
+    'dialogClass' => 'file-browser-container',
+    'width' => '90%',
+  );
+  $html = '';
+  $html .= theme('status_messages');
```

Comment:
Can we start this with `$html = theme('status_messages');` and remove `$html = '';` on the previous line?

Reply:
IMO I like starting with an empty string and then appending, it prevents me from making mistakes later if I rearrange the concatenations or add a new starting line.

Reply:
Fair enough :)